contracts/registry/NodeCrud.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity ^0.4.11;
+
+// This is the Node Create, Updated, Delete contract.
+contract NodeCrud {
```

Comment:
This contract is missing the state (attributes) and insert/update/retrieve methods.

Reply:
that's a pending task.